CSV/sample-entries.csv

```diff
@@ -52,6 +52,8 @@ hev2,High Efficiency Video Coding,Video,NALu Video,0x23
 hvc1,High Efficiency Video Coding,Video,NALu Video,0x23
 hvc2,High Efficiency Video Coding,Video,NALu Video,0x23
 hvt1,HEVC tile tracks,Video,NALu Video,
+hvt2,HEVC slice segment data track,Video,NALu Video,
+hvt3,HEVC Tile Track with Slice Segment Header Info,Video,NALu Video,
```

Comment:
I believe we should also add hev3 and hvc3, right? Everything else looks good to me.

Reply:
yes, added.